internal/kafka/internal/cmd/kafka/list.go

```diff
@@ -79,7 +79,10 @@ func runList(env *environments.Env, cmd *cobra.Command, _ []string) {
 	}
 
 	for _, kafkaRequest := range kafkaList {
-		converted := presenters.PresentKafkaRequest(kafkaRequest, kafkaConfig.BrowserUrl)
+		converted, err := presenters.PresentKafkaRequest(kafkaRequest, kafkaConfig)
+		if err != nil {
+			glog.Fatalf("Failed to format kafka request: %s", err.Error())
```

Comment:
Should this be a fatal error when iterating through the list? would it be better to log all the presenter issues for kakfa request items and then exit based on error or a bool value? Reason: we might exit after first error and would not unmask further errors down

Reply:
This is used in a CLI command so returning an error at the first occurrence seems reasonable to me